Source/projections/index.ts

```diff
@@ -1,63 +1,62 @@
 // Copyright (c) Dolittle. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-export { DeleteReadModelInstance } from './DeleteReadModelInstance';
-export { EventPropertyKeySelector } from './EventPropertyKeySelector';
-export { EventSelector } from './EventSelector';
-export { EventSourceIdKeySelector } from './EventSourceIdKeySelector';
-export { IProjection } from './IProjection';
-export { IProjections } from './IProjections';
-export { Key } from './Key';
-export { KeySelector } from './KeySelector';
-export { MissingOnMethodForType } from './MissingOnMethodForType';
-export { PartitionIdKeySelector } from './PartitionIdKeySelector';
-export { Projection } from './Projection';
-export { ProjectionCallback } from './ProjectionCallback';
-export { ProjectionContext } from './ProjectionContext';
-export { ProjectionId } from './ProjectionId';
-export { ProjectionResult } from './ProjectionResult';
-export { Projections } from './Projections';
-export { UnknownKeySelectorType } from './UnknownKeySelectorType';
-
-export * as internal from './Internal';
-
 export {
     CannotRegisterProjectionThatIsNotAClass,
     CouldNotCreateInstanceOfProjection,
     ICanBuildAndRegisterAProjection,
     KeySelectorBuilder,
     KeySelectorBuilderCallback,
-    OnDecoratedMethod,
-    OnDecoratedMethods,
     on,
+    OnDecoratedProjectionMethod,
+    OnDecoratedProjectionMethods,
+    OnMethodBuilder,
+    projection,
     ProjectionBuilder,
     ProjectionBuilderForReadModel,
     ProjectionClassBuilder,
     ProjectionClassOnMethod,
     ProjectionDecoratedType,
     ProjectionDecoratedTypes,
-    projection,
     ProjectionOptions,
     ProjectionsBuilder,
     ProjectionsBuilderCallback,
     ReadModelAlreadyDefinedForProjection,
+    TypeOrEventType
 } from './Builder';
-
+export { DeleteReadModelInstance } from './DeleteReadModelInstance';
+export { EventPropertyKeySelector } from './EventPropertyKeySelector';
+export { EventSelector } from './EventSelector';
+export { EventSourceIdKeySelector } from './EventSourceIdKeySelector';
+export * as internal from './Internal';
```

Comment:
Same as previous comment, shouldn't we just export things from here as we usually do